src/util-logopenfile.h

```diff
@@ -45,6 +45,9 @@ typedef struct LogFileCtx_ {
      * record cannot be written to the file in one call */
     SCMutex fp_mutex;
 
+    /** the type of file */
+    uint8_t type;
```

Comment:
Are you ok to move JsonOutput to OutputType and have it defined in util-logfile.h ?